src/SFA.DAS.FindEpao.Application.UnitTests/Courses/Queries/GetCourseEpao/WhenHandlingGetCourseEpaoQuery.cs

```diff
@@ -61,6 +61,7 @@ public class WhenHandlingGetCourseEpaoQuery
             result.DeliveryAreas.Should().BeEquivalentTo(courseEpao.DeliveryAreas);
```

Comment:
Does `StandardVersions` need to also be asserted?

Reply:
updated